codec/src/varint.rs

```diff
@@ -110,8 +110,9 @@ mod sealed {
         /// Compile-time assertion to ensure that the size of the signed integer is equal to the size of
         /// the unsigned integer.
         #[doc(hidden)]
-        const _COMMIT_OP_ASSERT: () =
-            assert!(std::mem::size_of::<Self>() == std::mem::size_of::<Self::UnsignedEquivalent>());
+        const _COMMIT_OP_ASSERT: () = assert!(
+            core::mem::size_of::<Self>() == core::mem::size_of::<Self::UnsignedEquivalent>()
```

Comment:
nit: my fault, but might be nice to just `use core::mem::size_of` instead of qualifying it everywhere